.github/PULL_REQUEST_TEMPLATE/new-pep.md

```diff
@@ -0,0 +1,72 @@
+---
+name: New PEP
+about: Submit a new draft PEP for consideration
```

Comment:
Good point, though a lot of it is still fully applicable—presumably, we should make another (shorter/more general) checklist for non-standards-track PEPs with the relevant elements. I can take a crack at it.

Reply:
After experimenting with several options—including separate Process/Information PEP checklist, and separate Process and Information checklists—I ended up re-organizing the existing new PEP checklist into a section that applied to all PEPs, and a section specific to Standards Track PEPs (that authors were instructed to remove if not applicable). I also caught a few other places (e.g. in the two other templates) where things only apply to Standards Track PEPs, and clarified them accordingly, as well as some other tweaks.